openshift/template.yaml

```diff
@@ -49,6 +49,8 @@ objects:
             value: "${DEFAULT_REPLICAS}"
           - name: MAX_REPLICAS
             value: "${MAX_REPLICAS}"
+          - name: OC_PROJECT
+            value: "${OC_PROJECT}"
```

Comment:
This needs to go to the parameters section as well: https://github.com/humaton/fabric8-analytics-scaler/blob/782d60505b5b395941e27edc7eb9a931ca41495c/openshift/template.yaml#L76

Reply:
fixed